autoload/neomake/makers/ft/chef.vim

```diff
@@ -4,11 +4,18 @@ endfunction
 
 function! neomake#makers#ft#chef#EnabledMakers() abort
     let ruby_makers = neomake#makers#ft#ruby#EnabledMakers()
-    return ruby_makers + ['foodcritic']
+    return ruby_makers + ['foodcritic'] + ['cookstyle']
 endfunction
 
 function! neomake#makers#ft#chef#foodcritic() abort
     return {
       \ 'errorformat': '%WFC%n: %m: %f:%l',
       \ }
 endfunction
+
+function! neomake#makers#ft#chef#cookstyle() abort
+    return {
+      \ 'args': [ '-f', 'emacs', '--except', 'Style'],
+      \ 'errorformat': '%f:%l:%c: %t: %m,%E%f:%l: %m',
```

Comment:
The first one should also be prefixed with `%E` probably?!
Should everything be an error really?